timer/Module/Intro/IntroViewCoordinator.swift

```diff
@@ -10,7 +10,7 @@
 class IntroViewCoordinator: CoordinatorProtocol {
      // MARK: route enumeration
     enum IntroRoute {
-        case timerSet
+        case timeSet
```

Comment:
I think `timeSet` is not proper name about presenting view.
Intro view will present main view that include local time set, productivity, shared time set views.
So intro route enumeration will have to `main` or specific main view (local, shared, productivity ...).

I suggest name `main` or `productivity` instead of `timeSet` .

Reply:
I will fix it.